src/neo4j_graphrag/llm/anthropic_llm.py

```diff
@@ -49,62 +58,74 @@ def __init__(
         self,
         model_name: str,
         model_params: Optional[dict[str, Any]] = None,
+        system_instruction: Optional[str] = None,
         **kwargs: Any,
     ):
-        try:
-            import anthropic
-        except ImportError:
+        if anthropic is None:
```

Comment:
What is the reason for this change? We had something similar in the past and moved to this new approach of importing the module locally.

Reply:
It's actually a silly reason, and let me know how I can handle it in a better way 😄 - I wanted to use the `MessageParam` type from the anthropic package as the return type for the `get_messages()` method (on ln 76), and this was the only way I could make it work 🤷‍♀️

Reply:
I was trying to follow the example from the already implemented method in the [BaseOpenAILLM class](https://github.com/leila-messallem/neo4j-graphrag-python/blob/bc6dd9c7b3f8fcfffb9ed360648ea80c6cbb17dc/src/neo4j_graphrag/llm/openai_llm.py#L62).

Would it be better to just change the return type to `list[dict[str, str]]`?

Reply:
To use a type from a dev dependency, you can do like this:

```
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from anthropic import MessageParam 
```

See how it's done in https://github.com/stellasia/neo4j-graphrag-python/blob/main/src/neo4j_graphrag/llm/openai_llm.py#L26

Reply:
To use a type from a dev dependency, you can do like this:

```
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from anthropic import MessageParam
```

Then MessageParam is available for mypy, and Python just ignores the annotations at runtime so it's not an issue.

Reply:
[6aea7fa](https://github.com/neo4j/neo4j-graphrag-python/pull/234/commits/6aea7fa31b056af58c75feb910f236397681c5d3)